src/api/chainHead.md

```diff
@@ -14,21 +14,21 @@ This beginner guide shows how to use the `chainHead` functions in order to know
 
 1. Call `chainHead_unstable_follow` with `withRuntime: true` to obtain a `followSubscription`. This `followSubscription` will need to be passed when calling most of the other `chainHead`-prefixed functions. If at any point in the future the JSON-RPC server sends back a `{"event": "stop"}` notification, jump back to step 1.
 
-2. When the JSON-RPC server sends back a `{"event": "initialized"}` notification with `subscription` equal to your `followSubscription`, store the value of `finalizedBlockHash` found in that notification.
+2. When the JSON-RPC server sends back a `{"event": "initialized"}` notification with `subscription` equal to your `followSubscription`, store the value of `finalizedBlockHashes` found in that notification. If `finalizedBlockHashes` contains multiple values, the user is encouraged to use the last one. This is the hash of the last block that has been finalized, for the purpose of this guide it will be named the `currentFinalizedBlock`. For all other values in `finalizedBlockHashes`, the user is encouraged to call `chainHead_unstable_unpin` with the `followSubscription`.
```

Comment:
Nit: personally I'd use the term _current finalized block_ instead of `currentFinalizedBlock` (and elsewhere); the latter looks like a reference to some variable in the code rather than defining a term.

```suggestion
2. When the JSON-RPC server sends back a `{"event": "initialized"}` notification with `subscription` equal to your `followSubscription`, store the value of `finalizedBlockHashes` found in that notification. If `finalizedBlockHashes` contains multiple values, the user is encouraged to use the last one. This is the hash of the last block that has been finalized, for the purpose of this guide it will be named the _current finalized block_. For all other values in `finalizedBlockHashes`, the user is encouraged to call `chainHead_unstable_unpin` with the `followSubscription`.
```

Reply:
I think I approved the PR without seeing the change in `chainHead.md`.

My remarks:

- The guide uses "you" everywhere. You shouldn't then say "the user".
- The "last" finalized block implies that no other block will ever be finalized in the future. You should always use "latest" instead of "last".
- Why "the user is encouraged"? The user must just use the last entry.
- Why say "for the purpose of this guide it's the current finalized block"? It **is** the current or latest finalized block, guide or not. The way it's phrased it just looks like you were too lazy to adjust the rest of the guide and delegate the effort to the reader.

Reply:
I agree with James that we shouldn't use `currentFinalizedBlock` with these quotes. Every ` used everywhere in the spec is used to refer to a field somewhere.